src/models/QueryStateModel.ts

```diff
@@ -92,7 +92,7 @@ export class QueryStateModel extends Model {
     quickview: 'quickview',
     debug: 'debug',
     numberOfResults: 'numberOfResults',
-    missingTerm: 'missingTerm'
+    missingTerms: 'missingTerms'
```

Comment:
had to change it, it was not the same in the `QUERY_STATE_ATTRIBUTES` at line 19 of this file